docs/10-WebServices.md

```diff
@@ -19,21 +19,43 @@ Configure modules in `api.suite.yml`:
 ``` yaml
 class_name: ApiTester
 modules:
-    config:
-		- REST:
-		    url: http://serviceapp/api/v1/
-		    depends: PhpBrowser
+    enabled:
+				- REST:
+				    url: http://serviceapp/api/v1/
+				    depends: PhpBrowser
 ```
```

Comment:
You should include module name and config manually. You just generate an empty suite, no one knows what modules you expect there